Makefile

```diff
@@ -1,13 +1,10 @@
-include Makefile.build
+include Makefile.build.mk
```

Comment:
This has bothered me for a long time, I like your change but how do you feel about `build.Makefile`.  I've slowly been doing this with Dockerfiles (aka okteto.Dockerfile)

Reply:
build.Makefile did not work for syntax highlighting